components/shared/workCard/WorkCard.tsx

```diff
@@ -84,6 +84,16 @@ const WorkCard = ({ work, className, isWithTilt = false }: WorkCardProps) => {
     },
     [] as typeof manifestationsWithPublizonData
   )
+
+  // filter out manifestations that not digital except physical books
```

Comment:
Wasn't there a similar function that you use somewhere else already that filters out the non-alowed types? Please use the same helper function where possible.